src/position/test/position.spec.js

```diff
@@ -16,18 +16,26 @@ describe('position elements', function () {
     $position = _$position_;
   }));
   beforeEach(function () {
-    this.addMatchers({
-      toBePositionedAt: function(top, left) {
-        this.message = function() {
-          return 'Expected "('  + this.actual.top + ', ' + this.actual.left +  ')" to be positioned at (' + top + ', ' + left + ')';
+    jasmine.addMatchers({
+      toBePositionedAt: function(util, customEqualityTesters) {
+        return {
+          compare: function(actual, top, left) {
+            var result = {
+              pass: util.equals(actual.top, top, customEqualityTesters) &&
+                      util.equals(actual.left, left, customEqualityTesters)
```

Comment:
It took me a bit of searching to figure this out, since it isn't on the introduction page for Jasmine.

Looking into it a little more, I think this is ok. My first instinct was that this should be changed to the simpler version, but in practice, users who are contributing wouldn't be modifying this. The Jasmine documentation in the source code mentions that this is a modification of Underscore's `isEqual` function at its heart.